how-to/privileged.md

```diff
@@ -0,0 +1,46 @@
+# Privileged Kata Containers
+
+Kata Containers supports creation of containers that are "privileged" - i.e. have additional capabilities and access
+that isn't normally granted.
+
+* [Caveats](#caveats)
+    * [Host Devices](#host-devices)
+        * [Containerd and CRI](#containerd-and-cri)
+
+## Warnings
+
+**Warning:** Whilst this functionality is supported, it can decrease the security of Kata Containers if not configured 
+correctly.
+
+### Host Devices
+
+By default, when privileged is enabled for a container, all of the `/dev/*` block devices from the host are mounted
```

Comment:
Suggest changing "all of" to more concise "all": 
...container, all the `/dev/*` block devices from the host are mounted